source/_integrations/symfonisk.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: IKEA SYMFONISK
+description: Connect and control your IKEA SYMFONISK devices using the Sonos integration
+ha_category:
+  - Media Player
+  - Sensor
+ha_domain: symfonisk
+ha_release: 0.7.3
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_domain: sonos
+ha_codeowners:
+  - '@cgtobi'
+  - '@jjlawren'
+ha_ssdp: true
+ha_platforms:
+  - binary_sensor
+  - diagnostics
+  - media_player
+  - number
+  - sensor
+  - switch
+ha_zeroconf: true
+ha_integration_type: integration
+ha_supporting_domain: sonos
+ha_supporting_integration: Sonos
```

Comment:
What's the difference between `ha_supporting_domain` and `ha_supporting_integration`?

Reply:
The capitalization? Name as shown in text vs. domain linked against.

Reply:
Ok.